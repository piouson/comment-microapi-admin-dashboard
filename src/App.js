import React from "react";
import { Admin, Resource } from "react-admin";
import Dashboard from "./components/dashboard/Dashboard";
import services from "./services";
import msadmins from "./components/msadmins";
import organizations from "./components/organizations";
import applications from "./components/applications";
import plans from "./components/plans";
import settings from "./components/settings";
import theme from "./theme";

function App() {
  return (
    <Admin theme={theme} dashboard={Dashboard} {...services}>
      <Resource name="Admins" {...msadmins} />
      <Resource name="Organizations" {...organizations} />
      <Resource name="Applications" {...applications} />
      <Resource name="Plans" {...plans} />
      <Resource name="Settings" {...settings} />
    </Admin>
  );
}

export default App;
