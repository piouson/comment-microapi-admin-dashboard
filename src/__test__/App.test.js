import * as React from "react";
import { shallow } from "enzyme";
import App from "../App";

test("Test App snapshot", () => {
  const app = shallow(<App />);
  expect(app).toMatchSnapshot();
});
