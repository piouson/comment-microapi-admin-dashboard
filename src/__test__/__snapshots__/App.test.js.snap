// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App snapshot 1`] = `ShallowWrapper {}`;
